developer-docs/latest/concepts/sso.md

```diff
@@ -0,0 +1,293 @@
+            }, (request, accessToken, refreshToken, profile, done) => {
+              done(null, {
+                email: profile.email,
+                firstname: profile.given_name,
+                lastname: profile.family_name,
+              });
+            }
+          ),
+        },
+      ],
+    },
+  },
+});
+```
+
+:::
+
+::: tab Github
+
+Using: [passport-github](https://github.com/cfsghost/passport-github)
+
+```bash
+yarn add passport-github2
+```
+
+`/config/server.js`
+
+```jsx
+'use strict';
+
+const GithubStrategy = require('passport-github2');
+
+module.exports = ({ env }) => ({
+  // ...
+  admin: {
+    // ...
+    auth: {
+      // ...
+      providers: [
+        {
+          uid: 'github',
+          displayName: 'Github',
+          icon: 'https://cdn1.iconfinder.com/data/icons/logotypes/32/github-512.png',
+          createStrategy: strapi => new GithubStrategy({
+            clientID: env('GITHUB_CLIENT_ID'),
+            clientSecret: env('GITHUB_CLIENT_SECRET'),
+            scope: ['user:email'],
+            callbackURL: strapi.admin.services.passport.getProviderCallbackUrl('github'),
+          }, (accessToken, refreshToken, profile, done) => {
+            done(null, {
+              email: profile.emails[0].value,
+              username: profile.username,
+            });
+          }),
+        },
+      ],
+    },
+  },
+});
+```
+:::
+
+::: tab Discord
+
+Using: [passport-discord](https://github.com/nicholastay/passport-discord#readme)
+
+```bash
+yarn add passport-discord
+```
+
+`/config/server.js`
+
+```jsx
+"use strict";
+
+const DiscordStrategy = require("passport-discord");
+
+module.exports = ({ env }) => ({
+  // ...
+  admin: {
+    // ...
+    auth: {
+      // ...
+      providers: [
+        {
+          uid: 'discord',
+          displayName: 'Discord',
+          icon: 'https://cdn0.iconfinder.com/data/icons/free-social-media-set/24/discord-512.png',
+          createStrategy: strapi =>
+            new DiscordStrategy(
+              {
+                clientID: env('DISCORD_CLIENT_ID'),
+                clientSecret: env('DISCORD_SECRET'),
+                callbackURL: strapi.admin.services.passport.getProviderCallbackUrl(
+                  'discord'
+                ),
+                scope: ['identify', 'email'],
+              },
+              (accessToken, refreshToken, profile, done) => {
+                done(null, {
+                  email: profile.email,
+                  username: `${profile.username}#${profile.discriminator}`,
+                });
+              }
+            ),
+        },
+      ],
+    },
+  },
+});
+```
+:::
+::::
+
+## Advanced Customization
+
+### Admin Panel URL
+
+If your administration panel lives on a different host/port than your Strapi server, you will need to specify the values in the admin config.
+To do so, heads to your `/config/admin.js` configuration file (create it if it doesn't exist) and tweaks the host and port fields.
```

Comment:
```suggestion
To do so, head to your `/config/admin.js` configuration file and tweak the host and port fields.
```

Here, to avoid having "create it if it doesn't exist" in parenthesis, you could turn this piece of information into a note.